src/bp-forums/forums/functions.php

```diff
@@ -2563,3 +2575,28 @@ function bbp_get_forums_per_page( $default = 15 ) {
 	// Filter and return
 	return (int) apply_filters( 'bbp_get_forums_per_page', $retval, $default );
 }
+
+/**
+ * Redirect non-loggedin user to login page while viewing private forum
+ *
+ * @param object $posts array of post objects.
+ * @param object $wp_query WP_Query inatsnace.
+ */
+function bbp_redirect_private_forum( $posts, &$wp_query ) {
```

Comment:
Newly introduced function should start with `bb_`

Reply:
Updated the function name.